src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/LocalizedPrefixMissingLocaleActionController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AttributeFixtures;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route(path: ['nl' => '/nl'])]
+class LocalizedPrefixMissingLocaleActionController
```

Comment:
Looks like we need to backport these two fixtures and fix the tests that use them. I'll have a look.

Reply:
see #51084

Reply:
merged up, rebase needed here now